hpack/definitions.yaml

```diff
@@ -0,0 +1,95 @@
+# This file defines sensible defaults but does not actually apply
+# any of them, so it can be safely included.
+
+_definitions:
+  _top-level:
+    - &meta
+        author:              Serokell, TQ Tezos
+        maintainer:          Serokell <hi@serokell.io>
+        copyright:           2019 TQ Tezos
+        license:             AllRightsReserved
+        license-file:        LICENSE
+        git:                 git@github.com:serokell/tezos-globacap.git
+        homepage:            https://github.com/serokell/tezos-globacap.git
+        bug-reports:         https://github.com/serokell/tezos-globacap/issues
+
+    - &default-extensions
+        - ApplicativeDo
```

Comment:
What's the point of enabling it here and disabling in the only file where this one is used?